.gitignore

```diff
@@ -44,3 +44,7 @@ yarn-debug.log*
 .DS_Store
 
 *.rdb
+
+# Ignore cypress assets
+/cypress/videos/*
+/cypress/screenshots/*
```

Comment:
nitpick: we don't need the wildcard (`*`) for these. We can ignore the directory itself.